.github/workflows/dbld-images.yml

```diff
@@ -3,17 +3,11 @@ name: Compile dbld-images
 on:
   pull_request:
   push:
-    paths:
-      - .github/workflows/dbld-images.yml
-      # List is from dbld/rules cache-image-% "WATCHED_FILES"
-      - "dbld/**"
-      - "packaging/rhel/syslog-ng.spec"
-      - "packaging/debian/control"
```

Comment:
For some reason I remembered that these paths were added to the `pull_request` trigger. Wouldn't they makes sense there even after your changes?

Reply:
I've checked the git history and I didn't find anything about pull requests and the above paths.

We could add those lines under the `pull_request` event type, if we would like to save some workflow run on PRs (i.e. only check dbld image build if dbld or packaging is changed).
As far as I understand GitHub pricing, we don't need to save because of that.

Another reason why we could limit this workflow to certain pull requests only is to avoid a CI failure due to an updated build-dependency. However since we don't use the newly built docker image for checking if syslog-ng can be compiled on it, I don't think that would be useful.

Reply:
I am worried about the build time, but that does not seem to be a problem right now, so It is fine by me, if it stays like this.